src/borg/archiver.py

```diff
@@ -1233,11 +1233,14 @@ def do_break_lock(self, args, repository):
 
         {now}
 
-            The current local date and time.
+            The current local date and time, by default in ISO-8601 format.
+            You can also supply your own format string, e.g. {now:%y}
```

Comment:
%y is the 2digit year, like 16 for 2016, which is a bit backwards since short before Y2K at least. ;)

Let's be not backwards and always use 4digit years, so it is %Y.